golos.charge/golos.charge.abi

```diff
@@ -135,12 +135,16 @@
             "type": "uint8"
         },
         {
-            "name": "stamp_id",
+            "name": "message_id",
```

Comment:
For charge smart-contract `message_id` is bad name. For send action back `useandnotify` must get `code:action` and some `id`, by which receiver can known for which record this answer (in case of publication it's message_id, in another case it's another identificator).
This action get two set of parameters: 1) as in `use` action (may be without last 2 parameters), and 2) `code, action, id` for callback.